packages/clerk-js/src/core/auth/AuthCookieService.ts

```diff
@@ -124,7 +127,7 @@ export class AuthCookieService {
         // be done with a microtask. Promises schedule microtasks, and so by using `updateCookieImmediately: true`, we ensure that the cookie
         // is updated as part of the scheduled microtask. Our existing event-based mechanism to update the cookie schedules a task, and so the cookie
         // is updated too late and not guaranteed to be fresh before the refetch occurs.
-        void this.refreshSessionToken({ updateCookieImmediately: true });
+        this.offlineScheduler.schedule(() => this.refreshSessionToken({ updateCookieImmediately: true }));
```

Comment:
we need to make sure wrapping this call doesn't impact the behavior described in the comment above.